src/parser/constants.ts

```diff
@@ -23,28 +26,50 @@ export const reFontDeclaration = new RegExp(
     '))?\\s+(.*)'
 );
 
-export const svgValidTagNames = [
-    'path',
-    'circle',
-    'polygon',
-    'polyline',
-    'ellipse',
-    'rect',
-    'line',
-    'image',
-    'text',
-  ],
-  svgViewBoxElements = ['symbol', 'image', 'marker', 'pattern', 'view', 'svg'],
-  svgInvalidAncestors = [
-    'pattern',
-    'defs',
-    'symbol',
-    'metadata',
-    'clipPath',
-    'mask',
-    'desc',
-  ],
-  svgValidParents = ['symbol', 'g', 'a', 'svg', 'clipPath', 'defs'],
+export enum ESVGTag {
+  path = 'path',
+  circle = 'circle',
+  polygon = 'polygon',
+  polyline = 'polyline',
+  ellipse = 'ellipse',
+  rect = 'rect',
+  line = 'line',
+  image = 'image',
+  text = 'text',
+}
```

Comment:
how enums move from TS to JS? i know const enum gets removed at compilation time, but those?

Reply:
If those Enums aren't really useful for the code flow, i prefer the static arrays